src/users/users.module.ts

```diff
@@ -5,9 +5,20 @@ import { TypeOrmModule } from '@nestjs/typeorm';
 import { User } from 'src/entities/user';
 import { EmailService } from './email.service';
 import { Email } from 'src/entities/email';
+import * as redisStore from 'cache-manager-redis-store';
+import type { ClientOpts } from 'redis';
+import { CacheModule } from '@nestjs/cache-manager';
 
 @Module({
-  imports: [TypeOrmModule.forFeature([User, Email])],
+  imports: [
+    CacheModule.register<ClientOpts>({
+      store: redisStore,
+      host: process.env.HOST,
```

Comment:
```suggestion
      host: process.env.REDIS_HOST,
```

`.env.sample`에 기재한 설명과 마찬가지로 `HOST`가 아닌 `REDIS_HOST`와 같은 명시적인 이름을 사용하면 좋을 것 같습니다~